packages/react-scripts/template/README.md

```diff
@@ -340,15 +341,15 @@ Next we add a 'lint-staged' field to the `package.json`, for example:
     // ...
   },
 + "lint-staged": {
-+   "src/**/*.{js,jsx,json,css}": [
++   "src/**/*.{js,jsx,ts,tsx,json,css}": [
 +     "prettier --single-quote --write",
 +     "git add"
 +   ]
 + },
   "scripts": {
 ```
 
-Now, whenever you make a commit, Prettier will format the changed files automatically. You can also run `./node_modules/.bin/prettier --single-quote --write "src/**/*.{js,jsx}"` to format your entire project for the first time.
+Now, whenever you make a commit, Prettier will format the changed files automatically. You can also run `./node_modules/.bin/prettier --single-quote --write "src/**/*.{js,jsx,ts,tsx}"` to format your entire project for the first time.
```

Comment:
We should probably add `json` and `css` to this initial command while we're here.

Reply:
@Timer  Here is all extensions supported by prettier: `css|less|scss|html|ts|tsx|graphql|gql|json|js|jsx|md`

Any other to add? I see cra added supported for saas recently.

https://github.com/prettier/prettier/blob/1244729ad74df56dd3b607dd395f0070c5e8baa6/.pre-commit-hooks.yaml#L6

Reply:
Heh, I wish there was a way to say "all files you can fix", can we recommend `pretty-quick`?